testing/architecture-test/src/changes/accepted-changes/exec-task-public-api-changes.json

```diff
@@ -0,0 +1,15 @@
+{
+    "acceptedApiChanges": [
+        {
+            "type": "org.gradle.api.tasks.Exec",
+            "member": "Class org.gradle.api.tasks.Exec",
+            "acceptation": "Invalid report for removed method AbstractExecTask.setIgnoreExitValue, since it's also upgraded",
+            "changes": [
+                "org.gradle.api.tasks.AbstractExecTask.setErrorOutput(java.io.OutputStream)",
+                "org.gradle.api.tasks.AbstractExecTask.setIgnoreExitValue(boolean)",
+                "org.gradle.api.tasks.AbstractExecTask.setStandardInput(java.io.InputStream)",
+                "org.gradle.api.tasks.AbstractExecTask.setStandardOutput(java.io.OutputStream)"
+            ]
+        }
```

Comment:
❓ Why don't we detect these changes automatically?

Reply:
I've opened https://github.com/gradle/gradle/pull/31461